Overviews/lightweightm2m_overview.html

```diff
@@ -75,8 +79,11 @@ <h2>Document File Types</h2>
             </ul>
 
             <h3><a href="/release/LightweightM2M/"><strong>Documents for download</strong></a></h3>
-
+       
             </div>
+        
+         <!-- until here -->
```

Comment:
Rather than 'until here' should it be more specific i.e 'DMSE WG content updates stop here'

Reply:
Good suggestion. Please add the  comment to this PR.